src/libtexture/imagecache_pvt.h

```diff
@@ -590,6 +594,7 @@ struct TileID {
     int m_miplevel;            ///< MIP-map level
     short m_chbegin, m_chend;  ///< Channel range
     int m_colortransformid;    ///< Colorspace id (0 == default)
+    int m_padding = 0;         ///< Unused
```

Comment:
This should probably be evaluated separately, but maybe instead of adding 32-bits of padding, `m_miplevel` and `m_colortransformid` could maybe be changed to 16-bit to shave off an extra 8 bytes?

I'd be at least curious to know if that saves any measurable amount of time.

Reply:
Yeah I didn't necessarily wish to bundle that type of change into this PR but I also thought something similar. 

Unfortunately I don't have a build with all the recent color space additions to test performance. The performance was measured on v2.4.6, prior to the addition of `m_colortransformid`.

Reply:
We can save this for another pass, but it seems like a good idea to me.